docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -709,6 +709,18 @@ WITH (
 AS SELECT name, comment, regionkey FROM tpch.tiny.nation;
 ```
 
+(delta-lake-shallow-clone)=
+#### Shallow cloned tables
+
+The connector supports read and write operations on shallow cloned tables. Trino
+does not support creating shallow clone tables. More information about shallow
+cloning is available in the [Delta Lake
+documentation](https://docs.delta.io/latest/delta-utility.html#shallow-clone-a-delta-table).
+
+Shallow cloned tables let you test queries or experiment with changes to a table
+without duplicating data. Any changes made to shallow cloned tables only affect
```

Comment:
This leaves out the important "as long as they don’t touch the source data Parquet files." from the original docs and is therefore misleading .. I would leave the sentence out.